orderer/common/multichannel/registrar.go

```diff
@@ -516,6 +516,45 @@ func (r *Registrar) SwitchFollowerToChain(chainName string) {
 	logger.Infof("Created and started channel %s", cs.ChannelID())
 }
 
+// SwitchChainToFollower creates a follower.Chain from the tip of the ledger and removes the consensus.Chain.
+// It is called when an etcdraft.Chain detects it was evicted from the cluster (i.e. removed from the consenters set)
+// and halts, transferring execution to the follower.Chain.
+func (r *Registrar) SwitchChainToFollower(channelName string) {
+	r.lock.Lock()
+	defer r.lock.Unlock()
+
+	if _, chainExists := r.chains[channelName]; !chainExists {
+		logger.Infof("Channel %s already removed", channelName)
+		return
+	}
+
+	if _, followerExists := r.followers[channelName]; followerExists {
+		logger.Panicf("Programming error, follower already exists: %s", channelName)
```

Comment:
Hopefully this is a non-issue as this will never print, but my initial reading of this was a little confusing.  It's not an error that the follow exists, it's that the follower _and_ the consenter exist.

Reply:
That is correct, fixed.